pymongo/command_cursor.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         comment: Any = None,
     ) -> None:
         """Create a new command cursor."""
-        self.__sock_mgr: Any = None
+        self.__conn_mgr: Any = None
```

Comment:
Unfortunately motor uses this property so I think it needs to stay for now: https://github.com/mongodb/motor/blob/2c0570f701c7bfb04e3b158aff7f0136ac75b61c/motor/core.py#L1695